ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -62,80 +49,93 @@ const FAM_UPLOAD_FORMATS = [].concat(BASE_UPLOAD_FORMATS)
 FAM_UPLOAD_FORMATS[1] = { ...FAM_UPLOAD_FORMATS[1], formatLinks: [...FAM_UPLOAD_FORMATS[1].formatLinks, { href: 'https://www.cog-genomics.org/plink2/formats#fam', linkExt: 'fam' }] }
 
 
-const BaseBulkContent = React.memo(({ actionDescription, details, project, name, requiredFields, optionalFields, uploadFormats, exportConfig, blankExportConfig }) =>
-  <div>
-    <Container>
-      To {actionDescription}, upload a table in one of these formats:
-      <StyledTable>
-        <Table.Body>
-          {uploadFormats.map(({ title, ext, formatLinks }) =>
-            <TableRow key={title}>
-              <TableCell>
-                <BoldText>{title}</BoldText> ({formatLinks ? formatLinks.map(
-                  ({ href, linkExt }, i) => <span key={linkExt}>{i > 0 && ' / '}<a href={href} target="_blank">.{linkExt}</a></span>)
-                  : `.${ext}`})
-              </TableCell>
-              <TableCell>
-                {ext &&
+export const BaseBulkContent = React.memo(({ url, actionDescription, details, project, name, requiredFields, optionalFields, uploadFormats, exportConfig, blankExportConfig }) =>
+  <StyledForm>
+    <NoBorderTable basic="very" compact="very">
```

Comment:
Instead of putting this Table in a container that adds a margin and then making it very compact, maybe try changing `compact="very"` to plain `compact` or taking out the `compact` option all together